### ISSUE TITLE
Doing the same trick with emoji font

### ISSUE THREAD

**Issue by milosponj** [horihiro/azure-functions-puppeteer-node-cjkfont#1]
hey @horihiro I've tried to adjust your code to do the same with noto-emoji font:

https://github.com/googlefonts/noto-emoji/raw/main/fonts/NotoColorEmoji.ttf

But adapting the postinstall script to use this font didn't work for me. (there shouldn't be an unzip command)

Is using this font instead possible, and if so, could you help me out in how would I do that?

**Comment by horihiro** [comment]
Which page do you try to display with noto-emoji font?

**Comment by horihiro** [comment]
And let me check your code.

**Comment by milosponj** [comment]
https://github.com/milosponj/azure-functions-puppeteer-node-cjkfont

Here's a fork of your project with my weak attempt

**Comment by horihiro** [comment]
try `curl` with `-L` option in `postinstall.sh`.
https://github.com/milosponj/azure-functions-puppeteer-node-cjkfont/blob/master/postinstall.sh#L3-L3

**Comment by horihiro** [comment]
How is this?

```diff
#!/bin/sh

+ mkdir -p /tmp/fonts
+ curl -s "https://github.com/googlefonts/noto-emoji/raw/main/fonts/NotoColorEmoji.ttf" -L -o /tmp/fonts/NotoColorEmoji.ttf && \
- curl -s "https://github.com/googlefonts/noto-emoji/raw/main/fonts/NotoColorEmoji.ttf" -o /tmp/fonts/ && \
mkdir -p ./fonts && mv /tmp/fonts/*.ttf ./fonts
```

Sample page: https://emojipedia.org/people/
| Before | After |
| :-: | :-: | 
|![image](https://user-images.githubusercontent.com/4566555/162546197-76108101-896f-4438-90cd-846f7f19dbc9.png)|![image](https://user-images.githubusercontent.com/4566555/162546189-757daf35-0810-4e4d-9ec1-ed417a27250b.png)|

**Comment by milosponj** [comment]
That looks very promising but my deployment is failing with:


```
Using Node version:
v14.19.1

Using Npm version:
6.14.16

Running 'npm install --unsafe-perm'...

npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!

> puppeteer@5.2.1 install /tmp/zipdeploy/extracted/node_modules/puppeteer
> node install.js


Chromium (782078) downloaded to /tmp/zipdeploy/extracted/node_modules/puppeteer/.local-chromium/linux-782078

> puppeteer-functions@1.0.0 postinstall /tmp/zipdeploy/extracted
> sh ./postinstall.sh

npm ERR! code ELIFECYCLE
npm ERR! errno 23
npm ERR! puppeteer-functions@1.0.0 postinstall: `sh ./postinstall.sh`
npm ERR! Exit status 23
npm ERR! 
npm ERR! Failed at the puppeteer-functions@1.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/.npm/_logs/2022-04-09T00_11_31_[59](https://github.com/milosponj/azure-functions-puppeteer-node-cjkfont/runs/5893261509?check_suite_focus=true#step:4:59)0Z-debug.log
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!\n\nnpm ERR! code ELIFECYCLE\nnpm ERR! errno 23\nnpm ERR! puppeteer-functions@1.0.0 postinstall: `sh ./postinstall.sh`\nnpm ERR! Exit status 23\nnpm ERR! \nnpm ERR! Failed at the puppeteer-functions@1.0.0 postinstall script.\nnpm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n\nnpm ERR! A complete log of this run can be found in:\nnpm ERR!     /home/.npm/_logs/2022-04-09T00_11_31_590Z-debug.log\n/opt/Kudu/Scripts/starter.sh oryx build /tmp/zipdeploy/extracted -o /home/site/wwwroot --platform nodejs --platform-version ~14
Deployment Failed.
Remote build failed!
Error: Process completed with exit code 1.
```

**Comment by milosponj** [comment]
I'm missing the first line of your suggestion though, let me try again

**Comment by milosponj** [comment]
....aaand it works! Thanks a bunch!
